WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java

```diff
@@ -1145,6 +1147,10 @@ public void onSiteEditorsChanged(OnSiteEditorsChanged event) {
                         ? SiteUtils.GB_EDITOR_NAME : SiteUtils.AZTEC_EDITOR_NAME;
                 mDispatcher.dispatch(SiteActionBuilder.newDesignateMobileEditorAction(
                         new DesignateMobileEditorPayload(event.site, defaultEditor)));
+                AnalyticsUtils.trackWithSiteDetails(
+                        AppPrefs.isGutenbergDefaultForNewPosts() ? Stat.EDITOR_GUTENBERG_ENABLED
+                                : Stat.EDITOR_GUTENBERG_DISABLED,
```

Comment:
Note that if we indeed emit the `_DISABLED` event here, we will probably see a spike from the users that have upgraded to app version 13.0. Those users won't actually have actively disabled the block editor but rather, it's just going to be the app itself broadcasting the fact that the classic editor gets saved to the backend.

I'd prefer if didn't emit this to avoid confusion when reading the tracks but, it's only going to be a "one time" event so probably not a real issue. WDYT?

Reply:
Totally agree with you. https://github.com/wordpress-mobile/WordPress-Android/pull/10254/commits/e8f1f2844fa25b0121a2b4752b17df20990c2c04 does fix the problem.